tensorboard/plugins/graph/keras_util.py

```diff
@@ -258,13 +258,19 @@ def keras_model_to_graph_def(keras_layer):
             node_def.attr["keras_class"].s = keras_cls_name
 
         dtype_or_policy = layer_config.get("dtype")
-        # Skip dtype processing if this is a dict, since it's presumably a instance of
+        dtype = None
+        # If this is a dict, try and extract the dtype string from
+        # `config.name`; keras will export like this in some cases. If we can't
```

Comment:
Is it possible to describe a bit more in "what cases"?

Reply:
Or perhaps reference the change in keras-team/keras#19711 ?

Reply:
Figured it out: this is for all non-input layers. Changed the comment.